### PR TITLE
[SD-934] Use email template in reset password email

### PR DESCRIPTION
**Pull request by kaliszKrzysztof**

### CODE REVIEW

**Review by AleksandraKaminska — APPROVED** [review]

File: config/locales/en.yml [hunk 1]
```diff
@@ -7,8 +7,13 @@ en:
...
       reset_password_instructions:
-        instructions_1: "A request to reset your password has been made.\nIf you did not make this request, simply ignore this email.\n\nIf you did make this request just click the link below:"
-        instructions_2: "If the above URL does not work try copying and pasting it into your browser.\nIf you continue to have problems please feel free to contact us."
+        welcome: Hi %{email}
+        instructions_1: A request to reset the password for your %{store_name} account has been made.
+        instructions_2: Click the button below to reset it.
+        instructions_3: If you did not request a password change ignore this email or contact us.
+        button: Reset your password
+        thanks: Thank you,
+        store_team: "%{store_name} Team"
```
Comment (przemosk) [inline config/locales/en.yml:16]: maybe you should ommit this interpolation like two above to keep consistency 😉

Comment (kaliszKrzysztof) [reply config/locales/en.yml:16]: Without this interpolation I'm getting Syntax Error:
```
<Psych::SyntaxError: (.../spree_auth_devise/config/locales/en.yml): found character that cannot start any token while scanning for the next token at line 16 column 21>
```

Comment (przemosk) [reply config/locales/en.yml:16]: oh, here: https://github.com/spree/spree_auth_devise/pull/504/files#diff-37c9b835927e3c363588f172de9bb6faR10 working without interpolation 🤔

Comment (kaliszKrzysztof) [reply config/locales/en.yml:16]: Seems like `%` char cannot be at the beginning of the phrase without interpolation

Comment (przemosk) [reply config/locales/en.yml:16]: so, maybe we should stick to rubocop style guide: https://rubystyle.guide/#string-interpolation ?

**Review by przemosk — APPROVED** [review]

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**